src/main/java/org/mule/runtime/api/config/MuleRuntimeFeature.java

```diff
@@ -100,7 +100,7 @@ public enum MuleRuntimeFeature implements Feature {
    */
   HONOUR_RESERVED_PROPERTIES(
       "When enabled, reserved properties such as 'app.name' can't be overridden by global properties (overrides are be ignored).",
-      "MULE-17659", "4.4.0, 4.3.0", HONOUR_RESERVED_PROPERTIES_PROPERTY),
+      "MULE-17659", "4.4.0", HONOUR_RESERVED_PROPERTIES_PROPERTY),
```

Comment:
```suggestion
      "MULE-17659", "4.3.0", HONOUR_RESERVED_PROPERTIES_PROPERTY),
```
See [this](https://github.com/mulesoft/mule/blob/cffbc1618617c2d22608a6c131e7d0bd06babd6d/core/src/main/java/org/mule/runtime/core/internal/context/DefaultMuleContext.java#L1296).